index.html

```diff
@@ -2177,25 +2187,34 @@ <h2>Install guide</h2>
         <div class="col_5">
           <ol>
             <li>
-              If you plan to store your roms on a SD card...
-              <strong>Format your SD Card in Steam UI. SD Cards need to be on ext4 (or btrfs) to work on EmuDeck</strong
-              >. Then go into Desktop mode by pressing the STEAM button, Power -> Switch to Desktop
+              To use an SD Card,
+              <strong>format your SD Card in Game Mode on SteamOS. SD Cards need to be formatted as ext4 (or btrfs) to be compatible with EmuDeck</strong>.
             </li>
-            <li>Download the Installer down below, copy that file to your Deck's Desktop and run it.</li>
-            <li>When The installation finish you'll need to copy yor games and run an app called Steam Rom Manager.</li>
+            <li>Switch to Desktop mode. To switch to Desktop Mode, press the STEAM button, then the Power button, and select Switch to Desktop.</li>
```

Comment:
@rawdatafeel I understand you wanted to make it all really clear, but now its such a long list that's going to intimidate people. People is just going to follow the video, the list is more to give a feeling of "wow, only 5 steps, that's easy!" I'm ok if you change those five steps to make it more clear or fix some grammar, but please keep it short :)